Makefile

```diff
@@ -415,3 +415,16 @@ test-app-dev: tmp/.ekam-run
 	@cp src/sandstorm/test-app/test-app.capnp tmp/sandstorm/test-app/test-app.capnp
 	@cp src/sandstorm/test-app/*.html tmp/sandstorm/test-app
 	spk dev -Isrc -Itmp -ptmp/sandstorm/test-app/test-app.capnp:pkgdef
+
+# ====================================================================
+# meteor-testapp.spk
+
+meteor-spk-0.3.2/meteor-spk:
+	@$(call color,downloading meteor-spk)
+	@curl https://dl.sandstorm.io/meteor-spk-0.3.2.tar.xz | tar Jxf -
+
+tests/assets/meteor-testapp.spk: meteor-testapp meteor-spk-0.3.2/meteor-spk $(shell find meteor-testapp -type f -o -type d)
+	@cd meteor-testapp && PATH="$$PATH:$$PWD/bin" && ../meteor-spk-0.3.2/meteor-spk pack ../tests/assets/meteor-testapp.spk
```

Comment:
No, I think this is because `spk` doesn't know where to find the capnp files. The installed spk finds the files by looking for the correct directory relative to itself, but now we're running the one that isn't installed, so it doesn't know where to look. To fix this, add `-I../src` to the `meteor-spk` command line -- this tells it to find the capnp files in `../src` (which contains `sandstorm/package.capnp`).